lib/sql.js

```diff
@@ -690,9 +690,24 @@ SQLConnector.prototype.update = function(model, where, data, options, cb) {
  */
 SQLConnector.prototype._replace = function(model, where, data, options, cb) {
   var stmt = this.buildReplace(model, where, data, options);
-  this.execute(stmt.sql, stmt.params, options, cb);
+  this.execute(stmt.sql, stmt.params, options, function(err, info) {
+    if (err) return cb(err);
+    if (info[0]) info = info.shift();
+    if (info.affectedRows === 0) {
+      return cb(errorIdNotFoundForReplace(where.id));
+    } else {
+      return cb(null, info);
+    }
+  });
 };
 
+function errorIdNotFoundForReplace(idValue) {
+  var msg = (g.f('Could not replace. Object with id %s does not exist!', idValue));
```

Comment:
Please remove the extra `()` around the expression, i.e. change `(g.f(...))` to `g.f(...)`